vignettes/manageSynapseCredentials.Rmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "Manage Synapse Credentials"
+author: "KimYen Ladia"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Manage Synapse Credentials}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+## Manage Synapse Credentials
+
+There are multiple ways one can login to Synapse. We recommend users to choose the method that fits your workflow.
+
+### One Time Login
+
+Use username and password to login as follow:
+```{r collapse = TRUE, eval = F}
+library("synapser")
+synLogin("username", "password")
+```
+
+### Use `.synapseConfig`
+
+For writing code using the `synapser` client, that easy to share with others, please do not include your credentials in the code. Instead, please use `.synapseConfig` file. You can prefer this documentation to your collaborators on how to setup `.synapseConfig` file.
```

Comment:
that IS easy to share
you can refer your collaborators to the following documentation on how to set up a .synapseConfig file